src/client/src/rt-platforms/openFin/adapter/openFin.ts

```diff
@@ -207,23 +220,19 @@ async function appRestoreHandler(workspaceApp: workspaces.WorkspaceApp, store: S
       await openDesktopWindow(
         config,
         () => {
-          store.dispatch(
-            LayoutActions.updateContainerVisibilityAction({
-              name: win.name,
-              display: true,
-            }),
-          )
+          callback({
+            name: win.name,
+            display: true,
+          })
         },
         { defaultLeft: win.bounds.left, defaultTop: win.bounds.top },
       )
 
       // we need to 'remove' the child window from the main window
-      store.dispatch(
-        LayoutActions.updateContainerVisibilityAction({
-          name: win.name,
-          display: false,
-        }),
-      )
+      callback({
```

Comment:
I don't know why you took this out of the store. Are you relying solely on openfin to save these layouts? if anything put it in localstorage. And I thought workspaces was just openfin. Why extract a platform specfic feature and generalize the naming?

Reply:
It wasn't taken out of the store, Bhavesh didn't want the entire store object passed to OpenFin adapter

Reply:
got it, alrighty then